msteams-platform/tabs/how-to/authentication/tab-sso-register-aad.md

```diff
@@ -5,13 +5,13 @@ ms.topic: how-to
 ms.localizationpriority: high
 keywords: teams authentication tabs Microsoft Azure Active Directory (Azure AD) access token SSO tenancy scope 
 ---
-# Register your tab app in Azure AD
+# Configure your tab app in Azure AD
 
 Azure AD provides access to your tab app based on the app user's Teams identity. You'll need to register your tab app with Azure AD so that the app user who has signed into Teams can be given access to your tab app.
 
 ## Enabling SSO on Azure AD
 
-Registering your tab app in Azure AD and enabling it for SSO requires making app configurations, such as generating app ID, defining API scope, and pre-authorize client IDs for trusted applications.
+Register your tab app in Azure AD and enable it for SSO requires making app configurations, such as generating app ID, defining API scope, and pre-authorize client IDs for trusted applications.
```

Comment:
NIT: If changing from the infinitive the rest of the sentence needs to be restructured. Right now, this is syntactically incorrect. 

Maybe 

> To enable your tab for SSO you must first configure the app, its scopes, and the pre-authorized client ids in Azure AD.